### ISSUE TITLE
Running in workspace is not support by semantic-release/npm

### ISSUE THREAD

**Issue by gsmlg** [gsmlg-dev/Foundation#12]
https://github.com/gsmlg-dev/Foundation/runs/5210807544?check_suite_focus=true

```
npm ERR! code ENOWORKSPACES
npm ERR! This command does not support workspaces.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/20[22](https://github.com/gsmlg-dev/Foundation/runs/5210807544?check_suite_focus=true#step:8:22)-02-16T03_31_07_772Z-debug-0.log
```

**Comment by gsmlg** [comment]
Resove with

```yaml
      - name: 🪓 Remove Workspace package.json
        working-directory: 'nodejs'
        run: rm package.json
```

**Comment by gsmlg** [comment]
fix at commit e9776670d4742d9407ba47638ac42a3f3406e513
